tests/commandline/test.py

```
@@ -294,30 +421,42 @@ def runTest():
-          expectedResult = t[2]
-        if len(t) > 3:
-          args = t[3]
-
-        if(os.sep == '\\' or os.name == 'nt'):
-            executable = (csoundExecutable == "") and os.path.join("..", "csound.exe") or csoundExecutable
+        expectedResult = (len(t) == 3) and 1 or 0
+
+        if os.sep == "\\" or os.name == "nt":
+            executable = (
+                (csoundExecutable == "")
+                and os.path.join("..", "csound.exe")
```

Ruff (0.14.1)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)